### ISSUE TITLE
Unhandled node type ConditionalExpression

### ISSUE THREAD

**Issue by nth10sd** [googleprojectzero/fuzzilli#410]
node parser.js ../../Protobuf/ast.proto code.js output.ast.proto

`code.js`:

```
1 ? 0 : 0;
```

Are ternary operators on latest rev d386a838 supported yet?

```
<repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:556
                throw "Unhandled node type " + node.type;
                ^
Unhandled node type ConditionalExpression
Thrown at:
    at visitExpression (<repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:556:17)
    at visitStatement (<repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:119:28)
    at visitProgram (<repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:51:37)
    at parse (<repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:561:12)
    at <repo>/Sources/Fuzzilli/Compiler/Parser/parser.js:570:15
    at finish (<repo>/Sources/Fuzzilli/Compiler/Parser/node_modules/protobufjs/src/root.js:105:9)
    at process (<repo>/Sources/Fuzzilli/Compiler/Parser/node_modules/protobufjs/src/root.js:143:13)
    at <repo>/Sources/Fuzzilli/Compiler/Parser/node_modules/protobufjs/src/root.js:195:17
    at fetchReadFileCallback (<repo>/Sources/Fuzzilli/Compiler/Parser/node_modules/@protobufjs/fetch/index.js:51:19)
    at readFileAfterClose (node:internal/fs/read/context:68:3)
```

**Comment by saelo** [comment]
Hi! The parser/compiler isn't feature complete, but we're happy to take pull requests! This is probably relatively easy to implement since we do [support ternary operators in the IL](https://github.com/googleprojectzero/fuzzilli/blob/d386a83812496b7e590ce93c8fdc50208dacbd26/Sources/Fuzzilli/FuzzIL/JsOperations.swift#L1424).

**Comment by nth10sd** [comment]
@TobiasWienand @saelo  Perhaps this is fixed by #438?

**Comment by saelo** [comment]
Right, this should indeed be fixed now!
